site/scripts/library.js

```diff
@@ -21,6 +21,11 @@ export function initializeLibrary() {
   let mangaItems = window.data.backupManga;
   const editCategoryOptions = document.getElementById("edit-category-options");
 
+  if (consts.fork.value === "sy")
```

Comment:
```suggestion
  if (consts.fork.value !== 'mihon')
```

Not a SY exclusive

Reply:
I see, don't use many forks so no clue. Do they handle the additional info in the same way?